src/utils/DataUtil.js

```diff
@@ -203,6 +207,20 @@ export class DataUtil {
       d.time = d.timestamp;
     }
 
+    if (d.type === 'cbg' && !d.sampleInterval) {
```

Comment:
This is basically the old `cgmSampleFrequency` utility transplanted here, with the `sampleInterval` being added to each cgm datum upon ingestion for legacy data that does not have it.

This is needed since we need to use a new crossfilter dimension to allow us to efficiently filter by `sampleInterval`, so each datum needs a value set.